app/src/main/java/org/simple/clinic/editpatient/EditPatientScreen.kt

```diff
@@ -181,6 +182,17 @@ class EditPatientScreen(context: Context, attributeSet: AttributeSet) : Relative
   }
 
   private fun setInputFieldsHint() {
+    val allGendersRadioButtons = mapOf(
```

Comment:
Since this map is being used in two different methods, what do you think of extracting it out of the method and pass it as a parameter to both the methods?

Reply:
Oh no, that's a mistake. I forgot to remove this in `setInputFieldsHunt`. I moved this logic into a separate method.